code/modules/codex/entries/clothing_codex.dm

```diff
@@ -79,6 +79,12 @@ GLOBAL_LIST_INIT(string_equip_flags, list("suit slot" = ITEM_SLOT_OCLOTHING,
 	if(!slowdown)
 		armor_strings += "<br>It will not slow the wearer down."
 
+	if(tint)
+		armor_strings += "<br>This will obstruct your vision."
+
+	if(accuracy_mod)
+		armor_strings += "<br>This will alter your shooting accuracy by [accuracy_mod]% when worn."
```

Comment:
This is not entirely accurate.
All the modifiers are summed up, and the result is a % reduction.
Maybe this?
```suggestion
		armor_strings += "<br>This will alter your shooting accuracy by up to [accuracy_mod]% when worn."
```